src/Poseidon/CLI/Init.hs

```diff
@@ -32,7 +35,7 @@ runInit (InitOptions format genoFile snpFile indFile outPath outName) = do
     let outInd = takeFileName indFile
         outSnp = takeFileName snpFile
         outGeno = takeFileName genoFile
-        genotypeData = GenotypeDataSpec format outGeno Nothing outSnp Nothing outInd Nothing
+        genotypeData = GenotypeDataSpec format outGeno Nothing outSnp Nothing outInd Nothing SNPSetOther
```

Comment:
Two options for `init`:

- We add an input field to let the user set the SNP set
- We find a clever way to determine the SNP set of the input genotype data automatically

Reply:
Hmm. I'm leaning towards having the user to input that via the command line.